app/controllers/api/v1/check_in_conferences_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Api::V1::CheckInConferencesController < ApplicationController
+  include SecuredPublicApi
+  before_action :set_profile
+
+  skip_before_action :verify_authenticity_token
+
+  rescue_from Pundit::NotAuthorizedError, with: :not_authorized
+
+  def create
+    @params = check_in_conference_params(JSON.parse(request.body.read, { symbolize_names: true }))
+    profile = Profile.find(@params[:profileId])
+    timestamp = Time.at(@params[:timestamp])
```

Comment:
rails では TimeWithZone が利用されるのでそちらに合わせてしまうのがよさそうかと
```suggestion
    timestamp = Time.zone.at(@params[:timestamp])
```

Reply:
👍